app/controllers/comments.rb

```diff
@@ -0,0 +1,9 @@
+post '/comments' do
+  @comment = Comment.new(params[:comment])
+  if @comment.save
```

Comment:
Associate new comments with the authenticated user, to prevent spoofing (so that I can't create resources as you):
```ruby
@comment = Comment.new(params[:comment])
@comment.user = current_user
if @comment.save
  # etc.
```